### PR TITLE
Add validation rule for russian language

### LINKED ISSUE #77: Add Russian and address properties for parser

### PR DESCRIPTION
**Pull request by IgorZiegler**
- add русский property for parser
 - add examples to modal.json

fixes #77

### CODE REVIEW

File: frontend/services/parser.js [hunk 1]
```diff
@@ -15,4 +15,6 @@ parser.registerProperty('длина', scope => {
   return value.length;
 });
 
+parser.registerProperty('русский', scope => /^[\u0400-\u04FF ]+$/.test(scope[scope.fieldName] || ''));
```
Comment (mvahrameev) [inline frontend/services/parser.js]: What about surnames such as 'Салтыков-Щедрин'?

Comment (102) [inline frontend/services/parser.js]: `д'Артаньян`

File: frontend/services/parser.js [hunk 2]
```diff
@@ -15,4 +15,10 @@ parser.registerProperty('длина', scope => {
   return value.length;
 });
 
+parser.registerProperty('русский', scope => /^[а-яА-ЯёЁ.\- ]{1,255}$/.test(scope[scope.fieldName] || ''));
+
+parser.registerProperty('адрес', scope => /^[0-9а-яёА-ЯЁ\\s\\.\\,\\-\\;\\/\\(\\)\\"]{1,255}$/.test(scope[scope.fieldName] || ''));
```
Comment (mvahrameev) [inline frontend/services/parser.js]: Return false on `7-ая Северная`

File: public/examples/forms/modal.json [hunk 3]
```diff
@@ -49,12 +59,20 @@
...
+          }]
         },
         {
           "block": "input",
           "placeholder": "дом",
-          "name": "building"
+          "name": "building",
+          "validationRules": [{
+            "rule": "дом",
+            "errorMessage": "Неверно введено отчество. Значение должно быть указано и состоять только из русских букв."
```
Comment (mvahrameev) [inline public/examples/forms/modal.json]: Strange error message...

File: frontend/services/parser.js [hunk 4]
```diff
@@ -15,4 +15,10 @@ parser.registerProperty('длина', scope => {
   return value.length;
 });
 
+parser.registerProperty('русский', scope => /^[а-яА-ЯёЁ.\- ]{1,255}$/.test(scope[scope.fieldName] || ''));
```
Comment (abukurov) [inline frontend/services/parser.js]: `/[а-яё]/i`

File: frontend/services/parser.js [hunk 5]
```diff
@@ -15,4 +15,10 @@ parser.registerProperty('длина', scope => {
...
+
+parser.registerProperty('адрес', scope => /^[0-9а-яёА-ЯЁ\\s\\.\\,\\-\\;\\/\\(\\)\\"]{1,255}$/.test(scope[scope.fieldName] || ''));
+
+parser.registerProperty('дом', scope => /^[0-9а-яА-Яa-zA-ZёЁ]{1,255}$/.test(scope[scope.fieldName] || ''));
```
Comment (abukurov) [inline frontend/services/parser.js]: 204/1?

File: frontend/services/parser.js (hunk 2 above)
Comment (abukurov) [inline frontend/services/parser.js]: `/^[0-9а-яёА-ЯЁ\\s\\.\\,\\-\\;\\/\\(\\)\\"]{1,255}$/.test('\s') === true`

**Comment by abukurov** [comment]
https://developer.mozilla.org/ru/docs/Web/JavaScript/Guide/Regular_Expressions

File: frontend/services/parser.js (hunk 5 above)
Comment (IgorZiegler) [inline frontend/services/parser.js]: This is corps not a house number.

File: public/examples/forms/modal.json (hunk 3 above)
Comment (mvahrameev) [inline public/examples/forms/modal.json]: Strange error message...

**Pull request merged**